QA_Robot_Practice/TestSuites/Clarabri-10001_PositiveLoginTest.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Documentation     A test suite with positive login tests.
+Suite Setup       Open login page
+Suite Teardown    Close Browser
+Resource         ../services/LoginService.robot
+Resource         ../utils/RandomWordsUtil.robot
+
+*** Variables ***
+${WELCOM_URL}     ${PROTOCOL}//${SERVER}:${PORT}/cmp/projects
+${ERROR_URL}      ${LOGIN_URL}/login?login_error=1
+${CREDENTIAL_SIZE}     5
+
+*** Test Cases ***
+Valid Credentials Test
+    Login To Clarabridge
+    Check Location     ${WELCOM_URL}
+    check main page content
```

Comment:
All words start with the first letter in Upper case. This comment for the rest of keywords